src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunClientSink.cs

```diff
@@ -22,11 +22,11 @@ public class RaygunClientSink : ILogEventSink
     private const string OccurredProperty = "RaygunSink_OccurredOn";
     private const string RaygunRequestMessagePropertyName = "RaygunSink_RequestMessage";
     private const string RaygunResponseMessagePropertyName = "RaygunSink_ResponseMessage";
+    private const string RaygunUserInfoPropertyName = "RaygunSink_UserInfo";
```

Comment:
Would be good to have this public, so that users don't need to use the string literal in their code when adding the user info key. The unit tests could then utilize this to avoid the duplicate too.